service/src/main/java/cz/muni/fi/pv254/parsing/App.java

```diff
@@ -196,9 +283,16 @@ public int inteligentParse(long gameID) {
             game = new GameDTO();
             game.setSteamId(gameID);
             game.setName(downloadGameName(gameID));
+            game.setShortDescription(downloadShortDescritpion(gameID));
             game = gameFacade.add(game);
         }
         game = gameFacade.findBySteamId(game.getSteamId());
+        Set<GenreDTO> genres = parseGenres(game);
+        game.setGenres(genres);
+        // TODO toto tu musi byt, inak ak hra nema ziadne reviews neulozia sa zanre
+        // TODO |
+        // TODO V
+        gameFacade.update(game);
```

Comment:
@VargaiD ak tu neurobim update tak ak hra nema ziadnu recommendation tak sa neulozi vazba genres:games
len ci to tu moze takto ostat

Reply:
Myslim ze ok